doc/tutorials/active_dendrite/nestml_active_dendrite_tutorial.ipynb

```diff
@@ -0,0 +1,595 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# NESTML active dendrite tutorial\n",
+    "\n",
+    "In this tutorial, we create a neuron model with a \"nonlinear\" or \"active\" dendritic compartment, that can, independently from the soma, generate a dendritic action potential. Instead of modeling the membrane potential of the dendritic compartment explicitly, the dendritic action potential (dAP) is modeled here as the injection of a rectangular (pulse shaped) dendritic current into the soma, parameterized by an amplitude and a duration. The rectangular shape can be interpreted as the approximation of an NMDA spike (Antic et al. 2010). A dendritic action potential is triggered when the total synaptic current exceeds a threshold.\n",
```

Comment:
As Tom suggested, I'd change the following sentence "the dendritic action potential (dAP) ... (pulse shaped) dendritic current" to "the dendritic action potential (dAP) is modeled here as the membrane-potential response to the injection of a rectangular (pulse shaped) dendritic current"

Reply:
I feel that this is another question of definition: the NMDA spike corresponds to the injection of the pulse-shaped current, and the resulting deflection in the membrane potential is just a downstream consequence of that; it is not part of the dAP itself, but controlled by other parameters such as C_m and R_m.